template.yaml

```diff
@@ -60,11 +74,26 @@ objects:
                 value: "7077"
               - name: SPARK_MASTER_WEBUI_PORT
                 value: "8080"
+              - name: NSS_WRAPPER_PASSWD
```

Comment:
You are right. But our development need the NSS_WRAPPER stuff also if the connect direct to a node. Especially if they start the spark shell. To prevent, that they must set the ENV everytime, i defined it global.